content/it/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,288 @@
+---
+reviewers:
+- caesarxuchao
+- dchen1107
```

Comment:
please remove the reviewers section. they're not needed as it will be the italian approvers/reviewers who are the reviewers